src/RageSoundReader_FileReader.cpp

```diff
@@ -50,7 +50,7 @@ RageSoundReader_FileReader *RageSoundReader_FileReader::TryOpenFile( RageFileBas
 		return nullptr;
 	}
 	OpenResult ret = Sample->Open( pFile );
-	pFile = nullptr; // Sample owns it now
+
```

Comment:
Setting pFile to nullptr ensures that anybody erroneously using it after this point runs into an obvious problem.  It's a common technique to make mistakes more visible.